### PR TITLE
Add hardcoded logo to football feed pages

### PR DESCRIPTION
**Pull request by kelvin-chappell**
This is the first iteration for getting sponsors' logos into football pages.  These pages have no contact with capi so cannot get logos in the conventional way.  Initially we are hardcoding them into the page instead.

This change also fixes a bug where these pages weren't targeting ads correctly.

/football/fixtures

Wide:
image

Mobile:
image

/football/live

Wide:
image

Mobile:
image

These pages don't work in Amp.

### CODE REVIEW

**Comment by kelvin-chappell** [comment]
[Virgin Media sponsorship of football fixtures and live scores pages](https://trello.com/c/SjthCMDc/115-virgin-media-sponsorship-of-football-fixtures-and-live-scores-pages)

File: sport/app/football/controllers/MatchListController.scala [hunk 1]
```diff
@@ -1,24 +1,28 @@
...
+import pa.FootballTeam
 import play.api.mvc.{Controller, RequestHeader}
-import common.{Edition, JsonComponent}
 import play.twirl.api.Html
-import implicits.Requests
-import pa.FootballTeam
 
 trait MatchListController extends Controller with Requests {
-  val competitionsService: Competitions
+  def competitionsService: Competitions
```
Comment (TBonnin) [inline sport/app/football/controllers/MatchListController.scala]: Why does this need to become a `def`?

File: sport/app/football/views/matchList/matchesPage.scala.html [hunk 2]
```diff
@@ -10,10 +16,16 @@
         <div class="content__main">
             <div class="gs-container">
                 <div class="content__main-column">
-                    <h2 class="hide-on-mobile-if-localnav content__inline-section page-heading--football">
+                    <h2 class="hide-on-mobile-if-localnav content__inline-section page-type--football">
                     @matchesList.pageType
                     </h2>
 
+                    <div class="page-sponsor--football">
+                    @for(commercial <- page.metadata.commercial; branding <- commercial.branding(Edition(request))) {
```
Comment (TBonnin) [inline sport/app/football/views/matchList/matchesPage.scala.html]: Can we have the statements on 2 lines? That's easier to read

**Review by TBonnin — COMMENTED** [review]
It shouldn't be to hard to bring CAPI to those controllers. 
Let me know if you want me to help

**Comment by paperboyo** [comment]
Logo from 10KB>2KB:

virgin_logo

File: sport/app/football/views/matchList/matchesPage.scala.html (hunk 2 above)
Comment (kelvin-chappell) [inline sport/app/football/views/matchList/matchesPage.scala.html]: Yes.  I'll change this.

**Comment by kelvin-chappell** [comment]
Thanks @paperboyo I'll use the new logo.

**Comment by kelvin-chappell** [comment]
@TBonnin There are some business issues with using capi.  There's currently no content in capi corresponding with these paths.  Logos are associated with tags (or sections) so there would need to be new keywords created specifically for these paths, to hold the sponsor's logo.  But central prod are reluctant to create new tags just to support a logo.

File: sport/app/football/controllers/MatchListController.scala (hunk 1 above)
Comment (TBonnin) [inline sport/app/football/controllers/MatchListController.scala]: Fair enough. Keep it in.

**Comment by TBonnin** [comment]
@kelvin-chappell Ok. I guess for a test it is fine to go for a workaround. We will need to find a proper way to do so if this become permanent.

**Comment by kelvin-chappell** [comment]
@TBonnin The main reason for putting a feature switch around it is with the hope of avoiding it becoming permanent!

**Pull request merged**